pom.xml

```diff
@@ -548,12 +548,13 @@ under the License.
         <artifactId>maven-surefire-plugin</artifactId>
         <version>2.22.1</version>
         <configuration>
+          <parallel>classes</parallel>
+          <useUnlimitedThreads>true</useUnlimitedThreads>
```

Comment:
If the number of classes is much more than the physical cores, would it cause performance regression?